.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   },
   rules: {
     'no-empty': 'off',
+    "object-curly-newline": 0,
```

Comment:
What is this for? It doesn't seem to cause any linting issues if I remove the line.

Reply:
Indeed, think it was something related to my environment. Can't get any linting diff when removing it. Removed the change.